iterative/utils/analytics_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func TestVersion(t *testing.T) {
-	assert.Equal(t, strings.HasPrefix(Version, "v"), true)
+	assert.Equal(t, "0.0.0", true)
```

Comment:
I would do something more flexible, indeed you can test that the ldflags works with 

```sh
go test -ldflags="-X 'terraform-provider-iterative/iterative/utils.Version=v3.0.0'" ./...
```

Reply:
A bit of defensive testing, perhaps? If we were using `gereleaser` for testing (hypothetically), it would make sense to check that the configuration is correct. Passing this flag in another place is equally prone to errors, and the only thing we're testing is that Go understands that flag. 🤯

Reply:
Voting to “resolve unresolved”

Reply:
> the only thing we're testing is that Go understands that flag

Not really, It helped me to figure out the right package structure to alter the Version. 
Not for automatised tests but for human checks... I agree that we might not needed it anymore but damn I was terribly helpful

Reply:
It would be helpful to check that `goreleaser` honors that option, but testing that is really, _really_ clunky. It may sound a bit lazy, but I would just omit this test. 🐊 

@iterative/cml, opinions?

Reply:
It fine, lets do not waste time on this.
I would have changed it ot check if version has three dots or something like that
But I guess its fine

Reply:
> _if version has three dots_

(Also known as, it's _exactly_ `0.0.0`)